src/main/java/nl/hsac/fitnesse/junit/HsacFitNesseRunner.java

```diff
@@ -101,6 +102,9 @@ protected FitNesseContext createContext(Class<?> suiteClass) throws Exception {
     protected void runPages(List<WikiPage> pages, RunNotifier notifier) {
         boolean seleniumConfigOverridden = configureSeleniumIfNeeded();
         try {
+            if ("true".equalsIgnoreCase(System.getProperty(allureEnablerVariableName))){
```

Comment:
This could (of course) also be done in the pom.xml (where I already add 2 listeners by default). Why does this need to happen in code (triggered by a system property)?